.rubocop.yml

```diff
@@ -42,3 +42,7 @@ SingleSpaceBeforeFirstArg:
 
 DotPosition:
   EnforcedStyle: trailing
+
+# Allow class and message or instance raises
+Style/RaiseArgs:
```

Comment:
See the changes - the error messages have now got too long for rubocop to be happy with having both of them in a message, so I've moved the main bits of the message out into the error classes themselves. I think it's nicer this way (the errors are single-use only anyway)